.circleci/config.yml

```diff
@@ -30,23 +30,48 @@ jobs:
           command: |
             python ./src/sync_dashboards/main.py sync --config lookml_dashboards.yaml
           no_output_timeout: 15m
+  validate-lookml:
+    executor: python/default
+    steps:
+      - checkout
+      - run:
+          name: Install spectacles
+          command: pip install spectacles
+      - run:
+          no_output_timeout: 20m
+          command: |
+            spectacles lookml \
```

Comment:
The docs say this only checks for syntax errors: https://docs.spectacles.dev/cli/tutorials/validators/#the-lookml-validator

Would the issue Teon mentioned be included in this?  I didn't look too deeply but intuitively it sounds like it be caught by the content validator

Reply:
We reproduced Teons issue when we tested it and it did detect it.

Reply:
From what I could tell `spectacles` is just calling the Looker API. And it seems it uses the lookml validation that is also part of the web UI

Reply:
Alright, good enough for me